apps/challenger/package.json

```diff
@@ -10,7 +10,7 @@
   "author": "daveroga",
   "license": "CC0-1.0",
   "dependencies": {
-    "@polygon-nightfall/common-files": "^0.0.5",
+    "@polygon-nightfall/common-files": "*",
```

Comment:
This needs corresponding changes in `package-lock.json` file.

Reply:
Very good catch, thank you!